connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -238,15 +239,21 @@ message StreamingQueryInstanceId {
   string run_id = 2;
 }
 
+message StreamingQueryInstance {
```

Comment:
Add a short comment. This name and `StreamingQueryInstanceId` might be confusing.
May be we should remove the `name` field completely and make it an API call to server when it is needed (in a follow up). 
For now, could you move it inside `StreamingQueryManagerCommand` as 'GetQueryResult`?